apps/mobile/src/features/stacks-tx-signer/stacks-tx-signer.tsx

```diff
@@ -47,7 +47,7 @@ interface ReviewTxSummaryProps {
 function formReviewTxSummary({ tx, symbol }: ReviewTxSummaryProps) {
   if (symbol !== 'STX') throw new Error('No support for SIP10');
 
-  // WARNING: Danerous type casting
+  // WARNING: Dangerous type casting
   const payload = tx.payload as TokenTransferPayloadWire;
```

Comment:
Quickly investigated this and perhaps would be better to do: 

```
  if (tx.payload.payloadType !== PayloadType.TokenTransfer) {
    throw new Error('Unsupported payload type');
  }
```
rather than typecast which is definitely not type safe in this instance.

Reply:
Sure, I can check it out but I just fixed a spelling error here in someone else's code